easybuild/easyconfigs/t/torchtext/torchtext-0.5.0-fosscuda-2019b-PyTorch-1.4.0--Python-3.7.4.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonPackage'
+
+name = 'torchtext'
+version = '0.5.0'
+local_pytorchver = '1.4.0'
+local_pyversuff = '-Python-%(pyver)s'
+versionsuffix = '-PyTorch-%s-%s' % (local_pytorchver, local_pyversuff)
+
+homepage = 'https://github.com/pytorch/text'
+description = "Data loaders and abstractions for text and NLP"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['7f22e24e9b939fff56b9118c78dc07aafec8dcc67164de15b9b5ed339e4179c6']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', local_pyversuff),
+    ('SentencePiece', '0.1.85', local_pyversuff),
+    ('tqdm', '4.41.1'),
+    ('PyTorch', local_pytorchver, local_pyversuff),
+]
+
+use_pip = True
+download_dep_fail = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/torchtext'],
```

Comment:
Isn't this the default in PythonPackage already?

Reply:
ok, removed.